addon/components/paper-switch.js

```diff
@@ -84,12 +92,13 @@ export default BaseFocusable.extend(RippleMixin, ProxiableMixin, ColorMixin, {
   },
```

Comment:
if you do that, you have to return a `SafeString` from the computed and use triple curlies.